api/resources_portal/migrations/0007_auto_20200407_1617.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 3.0.3 on 2020-04-07 16:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("resources_portal", "0006_auto_20200407_1455"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="material", name="needs_abstract", field=models.BooleanField(default=False),
+        ),
+        migrations.AlterField(
+            model_name="material",
+            name="category",
+            field=models.CharField(
+                choices=[
+                    ("CELL_LINE", "CELL_LINE"),
+                    ("PLASMID", "PLASMID"),
+                    ("PROTOCOL", "PROTOCOL"),
+                    ("DATASET", "DATASET"),
+                    ("MODEL_ORGANISM", "MODEL_ORGANISM"),
+                    ("PDX", "PDX"),
+                    ("OTHER", "OTHER"),
+                ],
+                max_length=32,
+            ),
+        ),
+    ]
```

Comment:
Can you delete these two migrations and regenerate them as one? Unless there's a particular reason these migrations can't be done together.

Reply:
Regenerated.

Reply:
Done.